universal-application-tool-0.0.1/test/auth/ProfileMergeTest.java

```diff
@@ -32,14 +36,26 @@ public UserRepository get() {
                 return repository;
               }
             });
+    samlProfileAdapter =
+        new SamlCiviFormProfileAdapter(
+            null,
```

Comment:
Can you inline comment these nulls per the style guide which says to doc ambiguous params at the call site.

/* paramName */ null

Reply:
done. I fixed the idcsProfileAdapter too.